### ISSUE TITLE
Regarding details of 'projector.py'

### ISSUE THREAD

**Issue by yunfan0621** [rosinality/stylegan2-pytorch#85]
Hi! Thank you for your great work!

I am recently working on embedding real images to the latent space using the code 'projector.py'. However, I am confused with line 133:

> latent_in = latent_mean.detach().clone().unsqueeze(0).repeat(2, 1)

What is the purpose of 'repeat(2,1)'? From my understanding, the latent code in w_plus space should have size [1, 512, 18], but I got [2, 512, 18] instead, and the reconstructed image is thus of size [2, 3, 1024, 1024].

And I also notice  that repeat() is not used in the test code in 'convert_weight.py'.

Thank you!

**Comment by rosinality** [comment]
Thank you, seems like that I made mistake. (Surprisingly code was runnable despite of this.) Fixed at 77f3aee.

**Comment by yunfan0621** [comment]
Glad that the code is further improved! Closing this issue~
